rmw_cyclonedds_cpp/src/rmw_node.cpp

```diff
@@ -3215,7 +3215,7 @@ extern "C" rmw_ret_t rmw_get_node_names_impl(
   }
 
   std::regex re {
-    "^name=([^;]*);namespace=([^;]*);(securitycontext=([^;]*);)?",
+    "^name=([^;]*);namespace=([^;]*);(enclave=([^;]*);)?",
```

Comment:
What is this for? Why didn't it use an underscore before?

Reply:
I believe that;s for user data.
looks like no underscore was used in any implementations 
https://github.com/ros2/rmw_connext/pull/407/files#diff-2469e9c52ef0878455668e9233e96b0aL79
https://github.com/ros2/rmw_fastrtps/pull/365/files#diff-9fc4d7c39bf46122c5317462daae5306L76

Reply:
>  What is this for? Why didn't it use an underscore before?

The others `rmw` implementations use a function to parse the key that don't accept a underscores: https://github.com/ros2/rmw/blob/6c513772fec7ccafe4eaaf5f838fbcfcc0d92393/rmw/include/rmw/impl/cpp/key_value.hpp#L70.

Using `enclave` instead of `security_context` here is correct.